src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,44 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $day-gap;
+  width: 7 * $day-size + 6 * $day-gap + 2 * $calendar-padding;
+  padding: 100px $calendar-padding;
+
+  &__day {
+    display: flex;
+    width: $day-size;
+    height: $day-size;
+    justify-content: center;
+    align-items: center;
+    border: 1px solid black;
+    background-color: $day-color;
+
+    transition: background-color 0.5s;
```

Comment:
You have made the `background-color` change smoothly, but the boxes are jumping up on hover. Maybe you should make them slide up smoother too. 

Please check this point `(Optional) both of these properties are to be animated with the duration of half a second.`

Reply:
Fair point by Andrei.
Yet, it's optional, so @huberciak you can smoothen `transform` in your spare time.